src/main/java/com/NewsWorkflw/common/controller/RssController.java

```diff
@@ -8,6 +8,8 @@
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.servlet.ModelAndView;
 
+//		is this a spring boot project? I cannot seem to run it and the springframework is one of the issues.

```

Comment:
Save this discussion for the pull request (where I am putting this comment; you can have inline conversations here).  It should not be in comments in the source code, because that's clutter, which is technical debt.